### PR TITLE
Evm events

### PR DESCRIPTION
**Pull request by matjazonline**

### CODE REVIEW

File: crawler/src/utils/utils.ts [hunk 1]
```diff
@@ -92,3 +94,12 @@ export const resolvePromisesAsChunks = async <T>(
 };
 
 export const removeUndefinedItem = <Type, >(item: (Type|undefined)): item is Type => item !== undefined;
+
+export const toContractAddress = (address: string): string =>{
```
Comment (Frenkiee) [inline crawler/src/utils/utils.ts:98]: use the one you wrote in API much cleaner 
```
export const toContractAddress = (address: string): string =>
  utils.getAddress(address.trim().toLowerCase());
```

File: db/sql/initialization/evmEvent.sql [hunk 2]
```diff
@@ -1,61 +1,99 @@
...
+CREATE INDEX IF NOT EXISTS evm_event_method ON evm_event (method);
+CREATE INDEX IF NOT EXISTS evm_event_event_id ON evm_event (event_id);
+CREATE INDEX IF NOT EXISTS evm_event_contract_address ON evm_event (contract_address);
+CREATE INDEX IF NOT EXISTS evm_event_topic_0 ON evm_event (topic_0);
+CREATE INDEX IF NOT EXISTS evm_event_topic_1 ON evm_event (topic_1);
+CREATE INDEX IF NOT EXISTS evm_event_topic_2 ON evm_event (topic_2);
+CREATE INDEX IF NOT EXISTS evm_event_topic_3 ON evm_event (topic_3);
+
+
+CREATE TABLE IF NOT EXISTS unverified_evm_call
```
Comment (Frenkiee) [inline db/sql/initialization/evmEvent.sql]: let's remove it if it is unused

File: db/sql/initialization/evmEvent.sql [hunk 3]
```diff
@@ -1,61 +1,99 @@
...
-      REFERENCES extrinsic(id),
-  CONSTRAINT fk_signer
-    FOREIGN KEY (signer)
-      REFERENCES account(address),
-  CONSTRAINT fk_verified_contract
-    FOREIGN KEY (contract_address)
-      REFERENCES verified_contract(address)
-);
\ No newline at end of file
+CREATE TABLE IF NOT EXISTS verified_evm_call
```
Comment (Frenkiee) [inline db/sql/initialization/evmEvent.sql]: also this one

File: crawler/src/utils/utils.ts (hunk 1 above)
Comment (matjazonline) [reply crawler/src/utils/utils.ts:98]: want to catch error here so we don't stop the explorer

**Pull request closed without merging**